### PR TITLE
[VC-38] - created home page

### PR DESCRIPTION
**Pull request by Voridan**

### CODE REVIEW

**Review by BoshkaS — COMMENTED** [review]
Everything is okay

**Review by IrynaPanchuk97 — CHANGES_REQUESTED** [review]
https://www.atlassian.com/git/tutorials/saving-changes/gitignore
Please determine which files needs to be ignored to avoid cluttering the repository.

File: VolunteerApp/ClientApp/src/app/signupBtn/signupBtn.component.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { Component } from '@angular/core';
+@Component({
+  selector: 'app-signup-button',
+  templateUrl: './signupBtn.component.html',
```
Comment (IrynaPanchuk97) [inline VolunteerApp/ClientApp/src/app/signupBtn/signupBtn.component.ts]: Please rename according to best practices

https://medium.datadriveninvestor.com/best-practice-naming-conventions-in-angular-af26430d2464

File: VolunteerApp/ClientApp/src/app/loginBtn/loginBtn.component.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-login-button',
+  templateUrl: './loginBtn.component.html',
```
Comment (IrynaPanchuk97) [inline VolunteerApp/ClientApp/src/app/loginBtn/loginBtn.component.ts]: Please rename according to best practices

https://medium.datadriveninvestor.com/best-practice-naming-conventions-in-angular-af26430d2464

File: VolunteerApp/ClientApp/src/app/loginBtn/loginBtn.component.css [hunk 3]
```diff
@@ -0,0 +1,20 @@
+.login {
+  display: inline-block;
+  color: #fff;
+  padding: 15px 20px;
```
Comment (IrynaPanchuk97) [inline VolunteerApp/ClientApp/src/app/loginBtn/loginBtn.component.css]: https://engageinteractive.co.uk/blog/em-vs-rem-vs-px

Please read Why you shouldn't “just use pixels”

File: VolunteerApp/ClientApp/src/app/header/header.component.ts [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    this.menuBody?.nativeElement.classList.toggle('menu-open');
+  }
+  closeMenu() {
+    document.querySelector('body')!.classList.remove('lock');
+    this.menuBtn?.nativeElement.classList.remove('menu-open');
+    this.menuBody?.nativeElement.classList.toggle('menu-open');
+  }
+
+  // @HostListener('window:resize', ['$event'])
+  // onWindowResize(){
```
Comment (IrynaPanchuk97) [inline VolunteerApp/ClientApp/src/app/header/header.component.ts]: Do not leave commented lines unnecessarily

File: VolunteerApp/ClientApp/src/app/app.module.ts [hunk 5]
```diff
@@ -1,34 +1,34 @@
 import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
-import { FormsModule } from '@angular/forms';
+// import { FormsModule } from '@angular/forms';
```
Comment (IrynaPanchuk97) [inline VolunteerApp/ClientApp/src/app/app.module.ts]: Please remove

NOTE:
Code is always better than comments. So always avoid comments where code can be made self explanatory.

**Review by Ostap-04 — APPROVED** [review]
All is well

**Review by BoshkaS — COMMENTED** [review]
ok

**Review by BoshkaS — COMMENTED** [review]
okay

**Review by BoshkaS — CHANGES_REQUESTED** [review]
okay

**Review by BoshkaS — APPROVED** [review]
okay

**Review by ykarpa — APPROVED** [review]

File: VolunteerApp/ClientApp/src/app/signupBtn/signupBtn.component.ts (hunk 1 above)
Comment (Voridan) [reply VolunteerApp/ClientApp/src/app/signupBtn/signupBtn.component.ts]: fixed

File: VolunteerApp/ClientApp/src/app/loginBtn/loginBtn.component.ts (hunk 2 above)
Comment (Voridan) [reply VolunteerApp/ClientApp/src/app/loginBtn/loginBtn.component.ts]: fixed

File: VolunteerApp/ClientApp/src/app/loginBtn/loginBtn.component.css (hunk 3 above)
Comment (Voridan) [reply VolunteerApp/ClientApp/src/app/loginBtn/loginBtn.component.css]: fixed

File: VolunteerApp/ClientApp/src/app/header/header.component.ts (hunk 4 above)
Comment (Voridan) [reply VolunteerApp/ClientApp/src/app/header/header.component.ts]: fixed

File: VolunteerApp/ClientApp/src/app/app.module.ts (hunk 5 above)
Comment (Voridan) [reply VolunteerApp/ClientApp/src/app/app.module.ts]: fixed

**Review by IrynaPanchuk97 — APPROVED** [review]

**Pull request merged**